packages/claim-cli/src/utils/build-account-list.ts

```diff
@@ -0,0 +1,57 @@
+import { NetworkParams } from './network-params';
+import { Account, CheckEligibilityResponse } from '../interfaces';
+import { ethers } from 'ethers';
+import L2ClaimAbi from '../abi/L2Claim';
+
+export interface AccountListChoice {
```

Comment:
May be it is better to define all interfaces in `interfaces.ts`.

Reply:
Moved this interface to `interface.ts`